src/sublime_aio.py

```diff
@@ -292,17 +318,18 @@ class AsyncEventListenerMeta(type):
     ```
     """
 
-    def __new__(mcs, name, bases, attrs):
+    def __new__(mcs: type[AsyncEventListenerMeta], name: str, bases: tuple[type, ...], attrs: dict[str, object]) -> AsyncEventListenerMeta:
```

Comment:
Does it make sense to wrap those long lines? This one even exceeds 120 chars limit.

```py
    def __new__(
        mcs: type[AsyncEventListenerMeta],
        name: str,
        bases: tuple[type, ...],
        attrs: dict[str, object]
    ) -> AsyncEventListenerMeta:
```

One arg per line also helps with readability, IMHO, especially if type hints are more complex.

Reply:
Yes, totally. I just forgot about this, probably.